facdb/sql/uscourts_courts.sql

```diff
@@ -0,0 +1,33 @@
+DROP TABLE IF EXISTS _uscourts_courts;
+SELECT
+    uid,
+    source,
+    buildingname as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    buildingaddress as address,
+    buildingcity as city,
+    zipcode,
+    NULL as boro,
+    NULL as borocode,
+    NULL as bin,
+    NULL as bbl,
+    courttype as factype,
+    (CASE
+		WHEN upper(courttype) LIKE '%COURT%' THEN 'Courthouses and Judicial'
+		ELSE 'Legal and Intervention Services'
+	END) as facsubgrp,
+    officename as opname,
+    NULL as opabbrev,
```

Comment:
This is a non-related question, but we need opabbrev to assign optype, we would have to think about how to deal with those cases where opabbrev is null, we might need a source specific update statement